roles/report_metadata_gen/README.md

```
@@ -0,0 +1,279 @@
+  Unsupported_CI_system0-->|Task| Fail_for_unsupported_CI_system1[fail for unsupported ci system]:::task
+  Fail_for_unsupported_CI_system1-->End
+```
+
+
+
+
+
+## Author Information
+Red Hat CI
+
+#### License
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)